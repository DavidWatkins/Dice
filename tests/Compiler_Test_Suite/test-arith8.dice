class test {
	public void main(char[][] args) {
		print(15+5.0);
		print("\n");
		print(1.5+1);
	}
}
