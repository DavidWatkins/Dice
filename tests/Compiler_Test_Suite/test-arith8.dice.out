20.000000
2.500000